app/ethos/management/commands/distribute_ethos.py

```diff
@@ -0,0 +1,69 @@
+'''
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+
+import json
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from ethos.models import Hop
+from gas.utils import recommend_min_gas_price_to_confirm_in_time
+from web3 import HTTPProvider, Web3
+
+w3 = Web3(HTTPProvider(settings.WEB3_HTTP_PROVIDER))
+
+
+class Command(BaseCommand):
+    help = 'distribute ethos tokens'
+
+    def handle(self, *args, **options):
+        nonce = w3.eth.getTransactionCount(settings.ETHOS_ACCOUNT_ADDRESS)
+        i = 0
+
+        for h in Hop.objects.all():
```

Comment:
You could eliminate the manual iteration of `i` by using `enumerate()` here.

Example:  `for i, h in enumerate(Hop.objects.all()):`